core-bundle/src/Command/InstallCommand.php

```diff
@@ -146,18 +152,18 @@ private function addEmptyDir(string $path): void
 
     private function addIgnoredDirs(): void
     {
-        static $ignoredDirs = [
-            'assets/css',
-            'assets/js',
+        $ignoredDirs = [
+            sprintf('%s/css', $this->assetsDir),
+            sprintf('%s/js', $this->assetsDir),
             'system/cache',
             'system/modules',
             'system/themes',
             'system/tmp',
-            '%s/share',
+            sprintf('%s/%s/share', $this->rootDir, $this->webDir),
         ];
 
         foreach ($ignoredDirs as $path) {
-            $this->addIgnoredDir($this->rootDir.'/'.sprintf($path, $this->webDir));
+            $this->addIgnoredDir($path);
```

Comment:
What about `system/modules` here? It is not converted to an absolute path anymore.